themes/university.typ

```diff
@@ -0,0 +1,263 @@
+// University theme
+//
+// By Pol Dellaiera - https://github.com/drupol
+//
+// Please feel free to improve this theme
+// by submitting a PR in https://github.com/andreasKroepelin/typst-slides
+//
+#let university-theme(
+    university-name: "University",
+    color-a: none,
+    color-b: none,
+    color-c: none,
+    logo: none
+) = data => {
+    let color-title = color-a
+    let color-subtitle = color-a
+    let color-section = color-a.lighten(85%)
+    let color-authors = black
+
+    let color-footer-fill-short-authors = color-a
+    let color-footer-fill-short-title = color-b
+    let color-footer-fill-date = color-b
+    let color-footer-fill-slide-counter = color-b
+
+    let color-footer-short-authors = white
+    let color-footer-short-title = white
+    let color-footer-date = white
+    let color-footer-slide-counter = white
+
+    let section = state("section", none)
+    let logical-slide = counter("logical-slide")
+    let subslide = counter("subslide")
+    let total-slides = counter("slide")
+    let last-slide-number = locate(loc => logical-slide.final(loc).first())
+
+    let title-slide = {
+        align(right)[
+            #block(
+            )[
+                #image(logo, width: 60mm)
```

Comment:
When the `logo` parameter is not provided, it defaults to `none`, which is a fine choice, I guess. However, you need to handle that case here. Code should not fail with the default argument.

Reply:
Fixed.